src/UI/templates/js/Core/ui.js

```diff
@@ -0,0 +1,51 @@
+var il = il || {};
+il.UI = il.UI || {};
+
+(function($, UI) {
+
+	UI.core = (function ($) {
```

Comment:
I would slightly prefer to have this on `UI` directly and also move this up one level. The other folders in /js refer to existing components, this doesn't.

Reply:
could also work. However, if we have many core functions in future, this approach would to lead many files on the top level or some few huge files. I would keep the core folder.